controller/argument_value_resolver.rst

```diff
@@ -239,6 +239,7 @@ priority of 100. As this one is responsible for fetching attributes from the
 lower priority. This makes sure the argument resolvers are not triggered when
 the attribute is present. For instance, when passing the user along a
 sub-requests.
+However, if the injected parameter is an attribute of the request, it is necessary to set a priority of at least 100 otherwise the resolver will not be launched.
```

Comment:
```suggestion
However, if the argument is resolved from a request attribute, it is necessary to set a priority of at least 100 otherwise the resolver will not be executed.
```

?